common/web/gesture-recognizer/src/tools/recorder/src/recorder.js

```diff
@@ -0,0 +1,113 @@
+// This script allows user-interactive use of the user-testing oriented unit-test-resource objects.
```

Comment:
To make a comparison with a previous PR (#6878), this, along with the relevant classes under `unit-test-resources`, is effectively an up-do on its `hostConfiguration.js`.  The caveat is that the backing script must be built first with this version, unlike that in #6878.

That said, it does allow us to do user-testing on classes that will see use _by_ upcoming unit testing, so that's pretty neat.  Well, it would be... if this were saved by our CI as a testing artifact.  But it's not in a good place for that.  (At least, not yet.)